pkgs/development/compilers/odin/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, fetchFromGitHub
+, llvmPackages
+, makeWrapper
+, git
+}:
+
+let
+  inherit (llvmPackages) stdenv;
+in stdenv.mkDerivation rec {
+  pname = "odin";
+  version = "0.13.0";
+
+  src = fetchFromGitHub {
+    owner = "odin-lang";
+    repo = "Odin";
+    rev = "v${version}";
+    sha256 = "xMRoFCFRYAAWI3ia2iX3pmPnZdthrOZ3CIAi7OlS/8A=";
+    leaveDotGit = true;
```

Comment:
It would be preferable to remove the .git directory and patch the version number manually in.